### PR TITLE
fix "changeLogs" casing in documentation

### PR DESCRIPTION
**Pull request by tarek-bochkati**
fix changeLogs casing in documentation
was written all lowercased which was leading to errors while write PDSC files

Signed-off-by: Tarek BOCHKATI <tarek.bouchkati@st.com>

### CODE REVIEW

**Comment by jkrech** [comment]
Tarek, I am a bit confused. You have changed the casing/capitalization of changelog to changeLog not only in the documentation but also in the PACK.xsd 
I think it is consistent today but just different. Why do you insist on a captial L? 
If you look into a dictionary `changelog` is a single word and not `change log`.
What am I missing?

File: schema/PACK.xsd [hunk 1]
```diff
@@ -2095,7 +2095,7 @@
         <!-- Version numbers and release history -->
         <xs:element name="releases" type="ReleasesType" minOccurs="1" />
         <!-- ID based references to changelog information stored in text files, associated with components -->
-        <xs:element name="changeLogs" type="ChangelogsType" minOccurs="0" />
+        <xs:element name="changeLogs" type="ChangeLogsType" minOccurs="0" />
```
Comment (tarek-bochkati) [inline schema/PACK.xsd:2098]: @jkrech the issue is that `changeLogs` is with a capital L in the .xsd file
the change in the xsd is just a cosmetic change to align with `LicenseSetsType` for `licenseSets`
so if the name is `changeLogs` (with capital L) I expected that the type is `ChangeLogsType` (with capital L) not `ChangelogsType`

Comment (jkrech) [reply schema/PACK.xsd:2100]: Wouldn't it be better to change the PACK.xsd then? 
Again my argument was that "changelog" is a single word and I cannot see a reason at this point why we should have a capital "L" in there. What do you think. 
Have you used it much yet?

Comment (tarek-bochkati) [reply schema/PACK.xsd:2100]: I though changing `changeLogs` to `changelogs` in the xsd is a breaking change
so I avoided doing so, do you prefer that I align the xsd to the documentation instead ?

Comment (tarek-bochkati) [reply schema/PACK.xsd]: done :)

**Review by jkrech — APPROVED** [review]
corrected spelling of changelog element.

**Comment by jkrech** [comment]
accepting not to update the version.

**Pull request merged**